crates/matrix-sdk-indexeddb/src/crypto_store/migrations.rs

```diff
@@ -281,6 +306,100 @@ fn migrate_stores_to_v7(db: &IdbDatabase) -> Result<(), DomException> {
     db.delete_object_store(old_keys::INBOUND_GROUP_SESSIONS_V1)
 }
 
+async fn migrate_data_for_v8(name: &str, serializer: &IndexeddbSerializer) -> Result<()> {
```

Comment:
there is a bit of a naming inconsistency here: `migrate_data_for_v6` is the data migration we do *after* upgrading the stores to v6. This should probably be called `migrate_data_for_v7`.

Reply:
That is quite confusing though, since we only introduced this migration when we introduced schema v8. Surely the important part of the version number is what version number triggers it, not whether it was before or after the schema change?

We could go with `migrate_data_after_v6` and `migrate_data_before_v8` ?

Reply:
> Surely the important part of the version number is what version number triggers it, not whether it was before or after the schema change?

I'm not entirely sure I follow you. Version number 6 (or earlier) triggers `migrate_data_for_v6`. Version number 7 triggers the new migration. And yes, the code for the new migration is being written a couple of weeks after v7 first landed but that doesn't seem entirely relevant.

Maybe it would be more intuitive if we named the data migrations after the schema version they are *preparing* for: ie, rename `migrate_data_for_v6` to `migrate_data_for_v7`?

Reply:
Yeah, that works for me.

Reply:
Done in 87e0fa017e